### PR TITLE
Fix Board to fill vertical space

### PR DESCRIPTION
**Pull request by drparcon**
Previously Board does not fill vertical space so this pull request fixes it so that the components fill the vertical space.

### CODE REVIEW

**Review by grantyap — APPROVED** [review]

File: src/routes/_layout.svelte [hunk 1]
```diff
@@ -15,5 +17,16 @@
     background-color: white;
     margin: 0 auto;
     box-sizing: border-box;
+    height: 100vh;
```
Comment (BigBlackLab19) [inline src/routes/_layout.svelte:20]: Dain, can you explain how 100vh solves the problem? How come 100% doesn't fix the job?

**Review by HazelArong — COMMENTED** [review]
Comment added

File: src/routes/_layout.svelte (hunk 1 above)
Comment (drparcon) [reply src/routes/_layout.svelte:20]: 100vh is 100% of the height of space depending on the device/browser while 100% height refers to the height of the container itself.

Comment (BigBlackLab19) [reply src/routes/_layout.svelte:20]: Got it, Dain! Thanks for the clarification

**Review by HazelArong — APPROVED** [review]

**Review by BigBlackLab19 — APPROVED** [review]

**Pull request merged**